frontend/package.json

```diff
@@ -8,36 +8,40 @@
     "dev-server": "webpack-dev-server"
   },
   "dependencies": {
+    "@babel/polyfill": "^7.10.4",
+    "@popperjs/core": "^2.4.4",
+    "@types/googlemaps": "^3.39.11",
     "autoprefixer": "^9.4.9",
     "axios": "^0.19.0",
     "babel-cli": "6.24.1",
     "babel-core": "6.25.0",
     "babel-loader": "7.1.1",
     "babel-plugin-transform-class-properties": "6.24.1",
     "babel-plugin-transform-object-rest-spread": "6.23.0",
+    "babel-polyfill": "^6.26.0",
     "babel-preset-env": "1.5.2",
     "babel-preset-react": "6.24.1",
     "bootstrap": "^4.3.1",
-    "css-loader": "^2.1.0",
     "firebase": "^7.15.5",
     "live-server": "^1.2.0",
-    "node-sass": "^4.11.0",
+    "node-sass": "^4.14.1",
     "postcss-loader": "^3.0.0",
     "precss": "^4.0.0",
     "rc-slider": "8.6.13",
-    "react": "^16.8.3",
     "react-bootstrap": "^1.0.0-beta.5",
     "react-dom": "^16.8.3",
     "react-google-maps": "^9.4.5",
     "react-redux": "5.1.1",
-    "react-router-dom": "4.1.2",
+    "react-router-dom": "^4.1.2",
     "redux": "4.0.1",
     "sass-loader": "^7.1.0",
     "style-loader": "^0.23.1",
     "uuid": "^3.3.3",
-    "webpack": "3.1.0"
+    "webpack": "^3.1.0",
```

Comment:
I'm also having a lot of errors like this:
```UnhandledPromiseRejectionWarning: TypeError: this.getResolve is not a function```
And then the process just hangs. Most searching ([stackoverflow](https://stackoverflow.com/questions/58546521/node-sass-and-sass-loader-module-build-failed-typeerror-this-getresolve-is-n)) says it's a compatibility issue between sass-loader and webpack but I think the versions we have should be okay, might be something else? Still trying to track that down.

Reply:
@chrisschaaf Yea, this is super frustrating error -- seems you're right this is a compatibility issue. Maybe we should also bump webpack to version 4+? If this is too much work, we we can just reject this PR and not bring in `@babel/polyfill` and the `async/wait` syntax.

Reply:
It's super strange though -- it was working fine last week and now I can't get it to build and serve the site :/

Reply:
I've been messing with it tonight, I think that's the right track. I definitely got it to build once during messing with this, but after a lot of updates I think I've got a stable set. Quick notes from what I had done:

> Babel refactors
	- `babel-` packages -> new `@babel/` packages (v6 -> v7/v8)
	- @babel/polyfill deprecated, using core-js/stable and regenerator-runtime/runtime https://babeljs.io/docs/en/babel-polyfill
	- babel-plugin-transform-* -> @babel/plugin-proposal-*
Updated rest to latest via `ncu -u` (npm-check-updates)
UUID import syntax:
	- `import uuid from 'uuid';` -> `import {v4 as uuid} from 'uuid';`
		○ default import was removed in current version. https://stackoverflow.com/questions/60830848/attempted-import-error-uuid-does-not-contain-a-default-export-imported-as-u
V4 (random) seemed like the most likely option, but v5 (generated from hashed namespace) may be a good option. May need to dicuss with MEC so they can coordinate?